ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -371,8 +371,20 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-upi
-- as: vsphere-ipi-ovn-ipsec-p2-f7
-  cron: 25 13 16,24,31,7 * *
+- as: vsphere-ipi-disconnected-p2-f14
+  cron: 41 6 4,19 * *
+  steps:
+    cluster_profile: vsphere
```

Comment:
need to use cluster_profile `vsphere-discon`

Reply:
fixed